### DISCUSSION TITLE
read your work with interest, visual scheduler

### DISCUSSION THREAD

**Discussion by ldijkman** [simeydotme/svelte-range-slider-pips#106] [Show and tell]
hello

used mrs slider
i am just a hobbyist
but i think the way selection and selected is drawn is not so nice
to much code generated
wich blocks me when i change  to minutes

trying to make a visual scheduler / graphical scheduler

https://plnkr.co/edit/0VOS27JydW81wwZc

https://ldijkman.github.io/randomnerd_esp32_wifi_manager/mrs/

https://github.com/ldijkman/randomnerd_esp32_wifi_manager/tree/main/docs/mrs

![2023-07-16-064919_1920x1080_scrot](https://github.com/simeydotme/svelte-range-slider-pips/assets/45427770/c48d11b9-c646-487a-8aa5-edbf0bc9b749)

**Comment by simeydotme** [comment]
apologies, I have no idea what your query is?
